src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -101,14 +105,16 @@ g.test('texture binding must have correct usage')
   ])
   .fn(async t => {
     const { type, _usage: usage } = t.params;
+    const info = kBindingTypeInfo[type];
 
+    const storageTextureFormat = info.kind === 'storage-texture' ? 'rgba8unorm' : undefined;
     const bindGroupLayout = t.device.createBindGroupLayout({
-      entries: [{ binding: 0, visibility: GPUShaderStage.FRAGMENT, type }],
+      entries: [{ binding: 0, visibility: GPUShaderStage.FRAGMENT, type, storageTextureFormat }],
     });
 
     const goodDescriptor = {
       size: { width: 16, height: 16, depth: 1 },
-      format: C.TextureFormat.R8Unorm,
+      format: C.TextureFormat.RGBA8Unorm,
```

Comment:
nit: already was this way, but use the string version?

Reply:
Oh wait, probably need this for type inference on goodDescriptor.

Reply:
Yup.